pages/common/npm-uninstall.md

```diff
@@ -0,0 +1,24 @@
+# npm uninstall
+
+> Remove a package.
+> More information: <https://docs.npmjs.com/cli/v8/commands/npm-uninstall>.
+
+- Remove a package from the current project:
+
+`npm uninstall {{package_name}}`
+
+- Remove a package globally:
+
+`npm uninstall -g {{package_name}}`
+
+- Remove multiple packages at once:
+
+`npm uninstall {{package_name1 package_name2 ...}}`
+
+- Remove a package and all its dependencies that are no longer needed:
+
+`npm uninstall {{package_name}} --save`
+
+- Remove a package and update `package.json`:
+
+`npm uninstall {{package_name}} --save-dev`
```

Comment:
Is this correct?

https://docs.npmjs.com/cli/v8/commands/npm-uninstall
> `--save` or `-S` will tell npm to remove the package from your `package.json`, `npm-shrinkwrap.json`, and `package-lock.json` files. This is the default


There's also no mention of `--save-dev` in the documentation.

Reply:
Then the example should be removed, since all previous commands cause `npm` to modify `package.json`, and this would suggest otherwise.